emanate/__init__.py

```diff
@@ -83,36 +78,22 @@ def del_symlink(self, path_obj):
 
         return not dest_file.exists()
 
-    def run(self):
-        all_files = Path(self.config.source).glob("**/*")
+    def get_files(self):
+        all_files = Path(".").glob("**/*")
```

Comment:
Using `Path(".")` again breaks non-CWD source support (and indeed breaks the test I wrote for it).